plugins/module_utils/vmware.py

```diff
@@ -218,3 +222,62 @@ def is_vcenter(self):
             return True
         elif api_type == 'HostAgent':
             return False
+
+    def get_obj(self, vimtype, name, return_all=False):
```

Comment:
i think this method's name should be more descriptive since you can also get VMs (and maybe other objects? im not sure) by UUID.
I called this `get_objs_by_name_or_moid` and thought you would call this method if you used name or moid, and call another method for UUID since its not applicable for all objs. Then theres a third more specific method that wraps these two and makes it easier for modules to search for things, like `get_vm`

What do you think?

Reply:
I will change it to `get_objs_by_name_or_moid`